### PR TITLE
Update community meetings page with latest meeting

### PR DESCRIPTION
**Pull request by Walavouchey**
wording copied from previous commits. still need agenda translations

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [ ] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)
  - [x] @DoryanR @Biokax (fr)
  - [x] @ZeroPyrozen (id)
  - [ ] @SiderealArt (zh-tw)
  - [x] @respektive (de)

### CODE REVIEW

**Comment by respektive** [comment]
German translations for
Line 11:
```md
Das nächste osu! Community Meeting wird zu einem späteren Zeitpunkt bekannt gegeben.
```
Line 42:
```md
| 11 | [20. Februar 2022](https://youtu.be/d66pU5lsHvE) | - | Mod-Multiplikatoren und Score |
```

File: wiki/Community/osu!_community_meetings/fr.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ Le premier osu! community meeting a eu lieu le 19 septembre 2021. Tous les meeti
 | 8 | [9 janvier 2022](https://youtu.be/JXgQ6YEDCGg) | [Document de synthèse](https://docs.google.com/document/d/1wJtJ7Agnsci3Ujxk52-ajeXfSJEKO-RCXDZCSUHcQYY) | Mécanique des HP (drain de santé), questions diverses de la communauté |
 | 9 | [22 janvier 2022](https://youtu.be/Prx0XzHl6-M) | [Document de synthèse](https://docs.google.com/document/d/1W_97ttbAo1mHjUgTeU_IB5SQVeQztT-pRrwiyTfjTu4) | Questions diverses de la communauté, mise à jour concernant le développement |
 | 10 | [6 février 2022](https://youtu.be/xA4nbE8DM4s) | [Document de synthèse](https://docs.google.com/document/d/1IM8LlHTrU9aIBkS-WTfbpLrMMrq2eRgRl7EAo_chDYE) | Questions diverses de la communauté |
+| 11 | [20 février 2022](https://youtu.be/d66pU5lsHvE) | - |  |
```
Comment (Ganondorf-1) [inline wiki/Community/osu!_community_meetings/fr.md:42]:
```suggestion
| 11 | [20 février 2022](https://youtu.be/d66pU5lsHvE) | - | Score et multiplicateurs de mods |
```

File: wiki/Community/osu!_community_meetings/id.md [hunk 2]
```diff
@@ -44,6 +44,7 @@ Pertemuan komunitas osu! pertama diadakan pada 19 September 2021. Semua pertemua
 | 8 | [9 Januari 2022](https://youtu.be/JXgQ6YEDCGg) | [Notulen pertemuan](https://docs.google.com/document/d/1wJtJ7Agnsci3Ujxk52-ajeXfSJEKO-RCXDZCSUHcQYY) | Mekanisme HP (*health drain*) yang berlaku di osu! serta berbagai pertanyaan dari para anggota komunitas |
 | 9 | [22 Januari 2022](https://youtu.be/Prx0XzHl6-M) | [Notulen pertemuan](https://docs.google.com/document/d/1W_97ttbAo1mHjUgTeU_IB5SQVeQztT-pRrwiyTfjTu4) | Berbagai pertanyaan dari para anggota komunitas serta berita-berita terkini seputar pengembangan osu! |
 | 10 | [6 Februari 2022](https://youtu.be/xA4nbE8DM4s) | [Notulen pertemuan](https://docs.google.com/document/d/1IM8LlHTrU9aIBkS-WTfbpLrMMrq2eRgRl7EAo_chDYE) | Berbagai pertanyaan dari para anggota komunitas |
+| 11 | [20 Februari 2022](https://youtu.be/d66pU5lsHvE) | - |  |
```
Comment (ZeroPyrozen) [inline wiki/Community/osu!_community_meetings/id.md:47]:
```suggestion
| 11 | [20 Februari 2022](https://youtu.be/d66pU5lsHvE) | - |  Pengali skor dan skor |
```

File: wiki/Community/osu!_community_meetings/en.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ Currently, the osu! community meetings are being chaired by [peppy](https://osu.
 
 Meetings are generally held every two weeks, with reminders announced on [peppy's Twitter](https://twitter.com/ppy) about a day beforehand.
 
-The next osu! community meeting is scheduled to take place on: **February 20, 2022 (Sunday 07:00 UTC)**.
+The next osu! community meeting date will be announced at a later date.
```
Comment (peppy) [inline wiki/Community/osu!_community_meetings/en.md:11]: I'm not really sure it's useful to include this and have to constantly update. Maybe just bold the "every two weeks" part above?

**Review by peppy — APPROVED** [review]

**Comment by peppy** [comment]
Merging as is so the link is added to the page.

**Pull request merged**